README.rst

```diff
@@ -49,3 +49,21 @@ Repository Overview
 .. _pypyjs-examples: https://github.com/pypyjs/pypyjs-examples
 .. _pypyjs.github.io: https://github.com/pypyjs/pypyjs.github.io
 .. _pypyjs.org: https://pypyjs.org
+
+Transpiling es6
+~~~~~~~~~~~~~~~
+
+Pypyjs is written using es6 and is transpiled to es5 using `babeljs`_ to run the
```

Comment:
We've been trying to use "PyPy.js" consistently when referring to the project; otherwise everyone makes up their own unique capitalization ;-)